src/background/hypothesis-chrome-extension.js

```diff
@@ -98,7 +104,7 @@ export default function HypothesisChromeExtension(dependencies) {
     }
 
     chromeTabs.create({ url: settings.serviceUrl + 'welcome' }, function (tab) {
-      state.activateTab(tab.id);
+      state.activateTab(/** @type {number} */ (tab.id));
```

Comment:
Here it is safe to assume the tab has an ID because we know it is a regular tab.